src/test/java/org/openmrs/performance/Constants.java

```diff
@@ -53,6 +53,12 @@ public class Constants {
 
 	public static final String ORDER = "39da3525-afe4-45ff-8977-c53b7b359158";
 
+	public static final String BACK_PAIN = "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+	public static final String ONSET_DATE = "2025-03-13T00:00:00+05:30";
+
+	public static final String PRACTITIONER_REFERENCE = "Practitioner/82f18b44-6814-11e8-923f-e9a88dcb533f";
```

Comment:
This uuid is the id of the logged in user, and it can be changed from implementation to implementation. Therefore, we can't hardcode it. Instead you should save it when the user logs into the system. It should be returned with the login request.